### PR TITLE
tobac

### PR DESCRIPTION
**Pull request by mheikenfeld**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mheikenfeld** [comment]
I am willing to act as a maintainer for the recipe

File: recipes/tobac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (xhochy) [inline recipes/tobac/meta.yaml]: Please remove the explanatory comments

File: recipes/tobac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/tobac/meta.yaml]: Delete empty build section

File: recipes/tobac/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
```
Comment (xhochy) [inline recipes/tobac/meta.yaml]: This can be `noarch: python`

File: recipes/tobac/meta.yaml (hunk 2 above)
Comment (mheikenfeld) [reply recipes/tobac/meta.yaml]: deleted

File: recipes/tobac/meta.yaml (hunk 1 above)
Comment (mheikenfeld) [reply recipes/tobac/meta.yaml]: I have removed all explanatory comments

File: recipes/tobac/meta.yaml (hunk 3 above)
Comment (mheikenfeld) [reply recipes/tobac/meta.yaml]: I've set noarch: python

File: recipes/tobac/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    - cf_units
+    - cartopy
+    - trackpy
+
+test:
+  imports:
+    - tobac
+    - tobac.testing
+    - numpy
+    - pandas
```
Comment (marcelotrevisani) [inline recipes/tobac/meta.yaml]: it is not necessary to import ``pandas`` and ``numpy``

Comment (mheikenfeld) [reply recipes/tobac/meta.yaml]: I have removed it from the test section.

File: recipes/tobac/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  summary: 'Tracking and Object-Based Analysis of Clouds'
+
+  description:  tobac is a flexible frameork for cloud tracking of different types of datasets
+  doc_url: http://tobac.readthedocs.io/
+  dev_url: https://github.com/climate-processes/tobac
+
+extra:
+  recipe-maintainers:
+    - mheikenfeld
+    - w-k-jones
```
Comment (marcelotrevisani) [inline recipes/tobac/meta.yaml]: @w-k-jones do you agree to be a maintainer of this recipe as well?

Comment (mheikenfeld) [reply recipes/tobac/meta.yaml]: He has agreed to me in a conversation yesterday that he'll be happy to act as a maintainer.

Comment (w-k-jones) [reply recipes/tobac/meta.yaml]: Just confirming that I'm happy to be a maintainer of this recipe as well

**Pull request merged**